Content.Server/FloofStation/Traits/Components/ProductiveCumProducerComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.FixedPoint;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FloofStation.Traits;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.GameStates;
+
+namespace Content.Server.FloofStation.Traits;
+
+[RegisterComponent, Access(typeof(LewdTraitSystem))]
+public sealed partial class ProductiveCumProducerComponent : Component
+{
+    [DataField("solutionname")]
+    public string SolutionName = "productivePenis";
```

Comment:
Use the "penis" solutionname since you can't have both productiveCum and cum -producer traits.